src/odemis/util/angleres.py

```diff
@@ -608,3 +608,68 @@ def _figure2data(figure):
 
     return image
 
+
+def ExtractThetaList(data):
+    """
+    Extracts the list of theta values so as to update the metadata MD_THETA_LIST.
+    Given the mirror parameters the theta list is calculated for ek imaging to be applied.
+    """
```

Comment:
Please add info about the input and output values. In particular, what are the minimum and maximum values returned, what is the length?